templates/detail.jinja2

```diff
@@ -1,16 +1,83 @@
 {% extends "base.jinja2" %}
 {% block body %}
 
+
+
+  <aside>
+    <form action="" method="POST" class="add_entry" id="edit">
```

Comment:
an odd discrepancy between the class name and the id here.  If it's for style purposes, perhaps make the class name less function-related?